data/rao-result/rao-result-api/src/main/java/com/powsybl/openrao/data/raoresultapi/io/Exporter.java

```diff
@@ -25,6 +25,8 @@ public interface Exporter {
      */
     String getFormat();
 
+    String getPropertiesPrefix();
```

Comment:
remove this from the interface and replace it with private methods in the implementations. you only need it inside the implementations, the user won't / mustn't use it